proto/autokitteh/connections/v1/connection.proto

```diff
@@ -2,11 +2,23 @@ syntax = "proto3";
 
 package autokitteh.connections.v1;
 
+import "autokitteh/common/v1/status.proto";
 import "buf/validate/validate.proto";
 
 message Connection {
   string connection_id = 1;
   string integration_id = 2 [(buf.validate.field).string.min_len = 1];
   string project_id = 3 [(buf.validate.field).string.min_len = 1];
   string name = 4 [(buf.validate.field).string.min_len = 1];
+
+  // Read only fields that are filled by the server.
+  common.v1.Status status = 5;
```

Comment:
why is this common and not connection status message ?
we already have codes for connect errors and sdk handling for errors

Reply:
this is not actually an error, but a status.

Reply:
is it used in somewhere else ?

Reply:
in connections and integrations